configs/machines/ecmwf-atos.yaml

```
@@ -0,0 +1,128 @@
+# THE ECMWF Atos YAML config file
+
+name: ecmwf-atos
+
+accounting: true
+batch_system: "slurm"
+
+operating_system: "linux-centos"
+
+jobtype: compute
+sh_interpreter: "/usr/bin/bash"
+
+
+partitions:
+        compute:
+                name: compute
+                cores_per_node: 128
+        pp:
+                name: gpil
+                cores_per_node: 1
+
+
+logical_cpus_per_core: 1
+
+threads_per_core: 1
+
+pool_dir: "/work/ollie/pool"
+
+pool_directories:
+        pool: "/work/ollie/pool"
+        projects: "/work/ollie/projects"
+        focipool: "/dev/null"
+
+# ?????
+choose_submitted:
+        true:
+                modules_needed:
+                        - centoslibs
+        false:
+                nothing: much
+
+submitted: false
+hyper_flag: "" # No hyperthreading on ollie
+choose_heterogeneous_parallelization:
+        True:
+                choose_taskset:
+                    False:
+                        add_export_vars:
+                            I_MPI_SLURM_EXT: 0
+                    "*":
+                        taskset_chs: ""
+                add_unset_vars:
+                        - "SLURM_DISTRIBUTION"
+                        - "SLURM_NTASKS"
+                        - "SLURM_NPROCS"
+                        - "SLURM_ARBITRARY_NODELIST"
+
+hetjob_flag: packjob
+
+unset_vars:
+        - "SLURM_MEM_PER_NODE"
+        - "SLURM_MEM_PER_CPU"
+
+# standard environment setup
+#
+#
+
+useMPI: openmpi
+module_actions:
+        - "reset"
+        - "load cmake"
+        - "load udunits"
+        - "load prgenv/intel"
```

Can you try moving the `prgenv/intel` to the top? It sets the context to avoid incompatibilities with compilers, so maybe we need this right at the top for later `module loads`?
```suggestion
        - "load prgenv/intel"
        - "load cmake"
        - "load udunits"
```

---

I made this change, but still the same issue of missing mpif.h.